qiskit/providers/ibmq/runtime/ibm_runtime_service.py

```diff
@@ -396,6 +396,16 @@ def delete_program(self, program_id: str) -> None:
         if program_id in self._programs:
             del self._programs[program_id]
 
+    def program_set_visibility(self, program_id: str, visibility: str) -> None:
+        """Sets a program's visibility to public.
+
+        Args:
+            program_id: Program ID.
+            visibility: the visibility of the program (public/private)
+
+        """
+        self._api_client.program_set_visibility(program_id, visibility)
```

Comment:
It'd be nice to convert `visibility` to all lower case before passing to `program_set_visibility` so users don't have to worry about casing.

Reply:
Fixed!